README.md

```diff
@@ -17,7 +17,7 @@ Community curated and predicted equivalences and related mappings between named
 that are not available from primary sources.
 
 Human-curated mappings are in `src/biomappings/resources/mappings.tsv`, and
-automatically predicted mappings are in `src/biomappings/resources/predictions.tsv`
+automatically predicated mappings are in `src/biomappings/resources/predictions.tsv`
```

Comment:
Huh, this is a typo I think I fixed on this branch that got added back

Reply:
I added it back to make merging easier

Reply:
Now there's no diff from this branch vs master for the README